aspnetcore/fundamentals/configuration/index.md

```diff
@@ -413,7 +479,7 @@ var config = new ConfigurationBuilder()
 
 The prefix is stripped off when the configuration key-value pairs are created.
 
-The static convenience method `CreateDefaultBuilder` creates a <xref:Microsoft.AspNetCore.Hosting.WebHostBuilder> to establish the app's host. When `WebHostBuilder` is created, it finds its host configuration in environment variables prefixed with `ASPNETCORE_`.
+The static convenience method `CreateDefaultBuilder` creates a host builder to establish the app's host. When host builder is created, it finds its host configuration in environment variables prefixed with `ASPNETCORE_`.
```

Comment:
Thanks @Tratcher ... my bad. Since I'm trying to keep this bit host-neutral, I'll link to where that detail is described ...

> When the host builder is created, host configuration is provided by environment variables. For more information on the prefix used for these environment variables, see the \[Default configuration](#default-configuration) section.